docs/modules/ROOT/pages/security-configuration.adoc

```diff
@@ -0,0 +1,28 @@
+= Security
+
+Security services are provided by the fabrication process to prevent access by unauthorized entities.  This is done
+through token based authentication.
+Each step in a pipeline is automatically wired up to validate an access tokens and determine if the provided identity
+is allowed to perform the action requested.
+
+== Configuring security services
+Security is configured through a property file located on the server which is handling the request.
+For example if you are running a machine learning pipeline you will find a property file on the server running the
+pipeline scripts.  The following is an example of a simple property file.
+
+.auth.properties
+[source,java]
+----
+# Properties file for model training authorization.
+#
+# GENERATED PROPERTIES FILE - please ***DO*** modify.
+
+pdp_host_url=http://policy-decision-point:8080/api/pdp
+is_authorization_enabled = True
+----
+
+When a new project is created security will be enabled by default and a policy-decision-point-docker module will be
+generated which needs to be added to your build process.
+If you decide to disable security you will need to change the configuration.
```

Comment:
A: This seems a little wordy.  To disable security update the 'auth.properties file and set 'is_authorization_enabled=False'.

Reply:
✅ AGREED & UPDATED